code-challenges.test.js

```diff
@@ -37,11 +63,28 @@ var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+// Create Jest test using syntax example from syllubus with describe, it, and expect statements.
+describe("capitalized", () => {
+    it("returns all of the strings in the array with the first letter capitalized", () => {
+      // set expect statement to equal the entire array with each string's first letter capitalized
+      expect(capitalized(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+      expect(capitalized(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
 
+    })
+  })
 
-// b) Create the function that makes the test pass.
-
+  // // Run the test to make sure you receive red error message that function name is not defined
+  // // Note: Jest test produced desired "red message"
 
+// b) Create the function that makes the test pass.
+// create function named capitalized
+const capitalized = (array) => {
+  // use .map() method to iterate over the entire array
+  // use .toUpperCase() method to upper case every 0'th index of each string in the array
+  // use .substring() method to return the rest of the string, after index 0
+  return array.map(value => value[0].toUpperCase() + value.substring(1, value.length))
+}
+// Note: function produces green "passed" message. Function refactored, as needed.
```

Comment:
Great job. Test looks fabulous. One note is that substring will default to the length of the string so you can actually remove that argument all together.